src/reducers-actions/orders.ts

```diff
@@ -40,9 +40,9 @@ export const reducer = (state: OrdersState, action: ReducerActionType): OrdersSt
 
       // default sorting order is ascending by order id
       // since we are starting from 0, reverse to have the latest on top
-      orders.reverse()
+      const reversed = orders.slice(0).reverse()
```

Comment:
You don't need `0` in `array.slice()`, not that it matters

Reply:
I read on [the internet this is faster](https://stackoverflow.com/a/21514254/1272513), so it must be true.

Reply:
Must be true then, no other way around it. Internet is the final authority

Reply:
Exactly! You get me.